pkg/tnf/interactive/spawner.go

```diff
@@ -286,10 +311,14 @@ func (g *GoExpectSpawner) Spawn(command string, args []string, timeout time.Dura
 	}
 
 	spawnFunc = (*spawnFunc).Command(command, args...)
-	stdinPipe, stdoutPipe, err := g.unpackPipes(spawnFunc)
+	stdinPipe, stdoutPipe, stderrPipe, err := g.unpackPipes(spawnFunc)
 	if err != nil {
 		return nil, err
 	}
+
+	cmdLine := strings.Join(args, " ")
```

Comment:
You're not using the Spawn()'s "command" argument, which is usually (but not always) "oc". I suggest changing this for something like:
cmdLine := fmt.Sprintf("%s %s", command, strings.Join(args, " "))

Reply:
done

Reply:
I thought args already included the command?

	// Args holds command line arguments, including the command as Args[0].
	// If the Args field is empty or nil, Run uses {Path}.
	//
	// In typical use, both Path and Args are set by calling Command.
	Args []string

Reply:
Ah, your change is in the setup phase where command and args are still separate as function args. I thought you were getting it from the SpawnFunc interface which wraps os.Cmd.Args().